apps/avatax/scripts/migrations/update-webhooks.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable turbo/no-undeclared-env-vars */
+
+import { AuthData } from "@saleor/app-sdk/APL";
+import { webhookMigrationRunner } from "@saleor/webhook-utils";
+import { createInstrumentedGraphqlClient } from "../../src/lib/create-instrumented-graphql-client";
+import { appWebhooks } from "../../webhooks";
+
+export const updateWebhooksScript = async ({
```

Comment:
I think we should remove `script` suffix. Everything is "script". And my first thought was "what is webhook script?"